DateInput/index.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { PropTypes } from 'react';
+import styles from './style.css'; // 'react-day-picker' uses global BEM-like classes
+import DayPicker from 'react-day-picker';
+
+const inThePast = (day) => {
+  let now = new Date();
+  return day < now;
+};
+
+const DateInput = ({
+  onChange,
+  scheduledAt,
+}) =>
+  <span>
+    <DayPicker
+      disabledDays={inThePast}
+      fromMonth={new Date()}
+      onDayClick={ (day, { disabled }) => {
+        if (disabled) return;
```

Comment:
If day is not marked as `disabled` allow `onChange` handler to be executed. 
`onChange` handler is passed in as prop -  will handle the updating of the store with the new selected day. 
Store change triggers re-render, which allows new selected day value to be passed in to `DateInput` as `scheduledAt` - causing newly selected day to be highlighted.